run.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Pass "dev" as an argument to run the dev server
+
+# rebuild container
+podman image build -t play-reading . ;
+
+# run the container
+podman run -v play_volume:/var/lib/postgresql/data \
+--name postgres-db \
+-e POSTGRES_PASSWORD=pass \
+-e POSTGRES_USER=playreading \
+-e POSTGRES_DB=playreadingparty \
+-p 5432:5432 \
+-d play-reading
+
+function cleanup {
+  echo "Removing postgres-db container"
+  podman stop postgres-db
+  podman rm postgres-db
+}
+trap cleanup EXIT
+
+
+if [ "$1" = "dev" ]; then
+  # start development server
+  NODE_ENV=development \
+  concurrently \
+    "webpack-dev-server --color" \
+    "nodemon --inspect ./src/server.js" \
+    "npm run css -- --watch --format beautify"
```

Comment:
Why have the -'-watch' and the '--format beautify' flags here and not in the script definition in package.json?

Reply:
Ah gorgeous, that makes a lot of sense. I had missed the addition of the `npm run css`  to the build script. 

As a side note, do we strictly need to have production have two scripts (build & start)  to get it going? Since we combine several commands to start the dev server on lines 29-31, could we not do the same for running build and start?

Reply:
https://github.com/rcheman/PlayReadingParty/pull/67